### PR TITLE
Update analog-output.md

### PR DESCRIPTION
**Pull request by topquark22**
## What This PR Changes
- (Please explain here why you created the pull request and specify what it changes)

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Comment by karlsoderby** [comment]
Hello @topquark22 , thank you for your submission. However, this only applies to some boards (particularly AVR boards). For example, the Nano RP2040 supports PWM on 20 pins (basically all pins).

File: content/learn/02.microcontrollers/03.analog-output/analog-output.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ In the graphic below, the green lines represent a regular time period. This dura
 
 ![](./pwm.gif)
 
+PWM is only available on digital output pins 3, 5, 6, 9, 10 and 11. The other digital pins cannot be used for PWM.
```
Comment (sebromero) [inline content/learn/02.microcontrollers/03.analog-output/analog-output.md:16]:
```suggestion
On some microcontrollers PWM is only available on selected pins. Please consider the pinout diagram of your board to find out which ones you can use for PWM. They are denoted with a tilde sign (~).
```

**Review by sebromero — CHANGES_REQUESTED** [review]
I made a proposal. Please let me know what you think.

**Comment by sebromero** [comment]
@karlsoderby What do you think about my suggestion?

File: content/learn/02.microcontrollers/03.analog-output/analog-output.md (hunk 1 above)
Comment (karlsoderby) [reply content/learn/02.microcontrollers/03.analog-output/analog-output.md:16]: @sebromero yes this works great, thanks. Let's see if in a later stage, we can add visuals to explain this better.

**Comment by sebromero** [comment]
@topquark22 I'm referring to the pinout diagrams that we provide. There we use the tilde symbols also for nano.

ABX00031-pinout

A000005-pinout

**Comment by topquark22** [comment]
Another thing you should mention is the frequency of the PWM signal. I measured it on my scope (from a Nano) and it was 976Hz.
G
----

    On Wednesday, December 7, 2022 at 10:24:43 a.m. EST, Sebastian Romero ***@***.***> wrote:  
 
 


@topquark22 I'm referring to the pinout diagrams that we provide. There we use the tilde symbols also for nano.

**Comment by sebromero** [comment]
I guess the PWM frequency again depends on the board, so I think I wouldn't add it in this generic article.

**Pull request merged**